.eslintrc.json

```diff
@@ -1,21 +1,46 @@
 {
-  "parser": "babel-eslint",
+  "parser": "@typescript-eslint/parser",
   "extends": ["airbnb", "prettier", "prettier/react"],
-  "env": { "jest": true },
+  "env": {
+    "jest": true
+  },
   "rules": {
-    "react/jsx-filename-extension": ["error", { "extensions": [".js"] }],
+    "react/jsx-filename-extension": [
+      "error",
+      {
+        "extensions": [".js", ".tsx"]
+      }
+    ],
     "import/no-extraneous-dependencies": [
       "error",
-      { "devDependencies": ["**/*.spec.js", "catalog/**/*", "config/**/*"] }
+      {
+        "devDependencies": ["**/*.spec.js", "catalog/**/*", "config/**/*"]
+      }
     ],
     "no-use-before-define": [
       "error",
       {
         "functions": false
       }
-    ]
+    ],
+    "import/extensions": [".js", ".jsx", ".json", ".ts", ".tsx"]
   },
   "globals": {
     "document": true
+  },
+  "overrides": [
+    {
+      "files": ["**/*.ts", "**/*.tsx"],
+      "rules": {
+        "no-unused-vars": ["off"]
```

Comment:
Do we need to disable this rule?

Reply:
At the moment yes. This rule will popup when importing interfaces. Because interfaces are not present once transpiled to typescript we are left with unused/unexisting imports. I can investigate this in another PR, but since it is not critical I feel that it can be done later. I'll keep a note on this to make more research.

Reply:
Thanks for the explanation. We can proceed with this rule disabled for `ts/tsx` files